core/src/test/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreTest.java

```diff
@@ -275,18 +271,19 @@ public void testDefaultGroupIdConfig() throws Exception {
   }
 
   @Test(expected=StoreInitializationException.class)
-  public void testMandatoryCompationPolicy() throws Exception {
+  public void testMandatoryCompactionPolicy() throws Exception {
     Properties kafkaProps = new Properties();
     Map<String, String> topicProps = new HashMap<>();
-    topicProps.put(LogConfig.CleanupPolicyProp(), "delete");
+    topicProps.put(TopicConfig.CLEANUP_POLICY_CONFIG, "delete");
 
     NewTopic topic = new NewTopic(SchemaRegistryConfig.DEFAULT_KAFKASTORE_TOPIC, 1, (short) 1);
     topic.configs(topicProps);
 
     Properties props = new Properties();
     props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
-    AdminClient admin = AdminClient.create(props);
-    admin.createTopics(Collections.singletonList(topic));
+    try (AdminClient admin = AdminClient.create(props)) {
+      admin.createTopics(Collections.singletonList(topic)).all().get(60000, TimeUnit.MILLISECONDS);
```

Comment:
Why not use 60 seconds? Also, should this be a constant to make things a bit more consistent?